src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,6 +105,27 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
```

Comment:
The direction key handlers should not execute if the new param is undefined (remember to add explicit checks for undefined params)